config/webpack.config.js

```diff
@@ -87,7 +88,8 @@ if (__DEV__) {
   debug('Enabling plugins for live development (HMR, NoErrors).')
   webpackConfig.plugins.push(
     new webpack.HotModuleReplacementPlugin(),
-    new webpack.NoErrorsPlugin()
+    new webpack.NoErrorsPlugin(),
+    new OpenBrowserPlugin({ url: project.compiler_public_path })
```

Comment:
Hey again,

Sorry for the confusion. I was using my own application based off of the starter-kit to come to this conclusion. Turns out, someone on my team had replaced the compiler_public_path in the environments.config.js file.

Sorry for the confusion!